src/Application/Results.php

```diff
@@ -5,14 +5,15 @@
 namespace JeroenG\Explorer\Application;
 
 use Countable;
+use Elastic\Elasticsearch\Response\Elasticsearch;
 
 class Results implements Countable
 {
     private array $rawResults;
 
-    public function __construct(array $rawResults)
+    public function __construct(array|Elasticsearch $rawResults)
     {
-        $this->rawResults = $rawResults;
+        $this->rawResults = $rawResults instanceof Elasticsearch ? $rawResults->asArray() : $rawResults;
```

Comment:
```suggestion
        $this->rawResults = is_array($rawResults) ? $rawResults : $rawResults->asArray();
```
this will probably make github action happier